spec/controllers/v2/reports_controller_spec.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe V2::ReportsController do
+  let(:attributes) do
+    {
+      title: :title,
+      os_major_version: :os_major_version,
+      ref_id: :ref_id,
+      description: :description,
+      profile_title: :profile_title,
+      business_objective: :business_objective,
+      all_systems_exposed: -> { true },
+      compliance_threshold: :compliance_threshold,
+      assigned_system_count: -> { 4 },
+      result_system_count: -> { 4 },
+      compliant_system_count: -> { 1 },
+      unsupported_system_count: -> { 2 }
+    }
+  end
+  before { stub_rbac_permissions(Rbac::INVENTORY_HOSTS_READ, Rbac::REPORT_READ) }
+
+  let(:current_user) { FactoryBot.create(:v2_user) }
+  let(:rbac_allowed?) { true }
+
+  before do
+    request.headers['X-RH-IDENTITY'] = current_user.account.identity_header.raw
+    allow(StrongerParameters::InvalidValue).to receive(:new) { |value, _| value.to_sym }
+    allow(controller).to receive(:rbac_allowed?).and_return(rbac_allowed?)
+  end
+
+  context '/reports' do
+    describe 'GET index' do
+      let(:extra_params) { { account: current_user.account } }
+      let(:parents) { nil }
```

Comment:
Is this really needed? I think you can omit it if tests won't fail. Maybe other tests have it wrongly too, we can clean that up later, but this one could go.

Reply:
Can be omitted :+1: